python/sglang/srt/server.py

```diff
@@ -521,17 +521,22 @@ async def async_generate(
         prompt: str,
         sampling_params: Optional[Dict] = None,
     ):
+        if isinstance(sampling_params, dict) and sampling_params.get("n", 1) > 1:
+            stream = False
+        else:
+            stream = True
```

Comment:
Revert this as it is supported in https://github.com/sgl-project/sglang/commit/d8476818efc88188d0aa0a8a176024a0b82e7a1d